packages/components/src/components/transfer-account-selector/transfer-account-selector.tsx

```diff
@@ -20,65 +20,69 @@ type TTransferAccountSelectorProps = {
-    }, [value]);
+        React.useEffect(() => {
+            setSelectedAccount(value);
+        }, [value]);
 
-    const openAccountsList = () => {
-        setIsListModalOpen(true);
-    };
+        const openAccountsList = () => {
+            setIsListModalOpen(true);
+        };
 
-    return (
-        <div className='transfer-account-selector' onClick={is_list_modal_open ? undefined : openAccountsList}>
-            <TransferTile
-                is_mobile={is_mobile}
-                label={label}
-                placeholder={placeholder}
-                selected_account={selected_account}
-            />
-
-            {is_mobile ? <div id='mobile_list_modal_root' /> : <div id='desktop_list_modal_root' />}
-
-            <Modal
-                className='transfer-account-selector__modal-header'
-                has_close_icon
-                is_open={is_list_modal_open}
-                portalId={portal_id}
-                title={label}
-                toggleModal={() => setIsListModalOpen(old => !old)}
-                min_height={is_mobile ? mobile_list_min_height : ''}
-            >
-                <TransferAccountList
+        return (
+            <div className='transfer-account-selector' onClick={is_list_modal_open ? undefined : openAccountsList}>
+                <TransferTile
                     is_mobile={is_mobile}
+                    label={label}
+                    placeholder={placeholder}
                     selected_account={selected_account}
-                    setIsListModalOpen={setIsListModalOpen}
-                    setSelectedAccount={setSelectedAccount}
-                    transfer_accounts={transfer_accounts}
-                    transfer_hint={transfer_hint}
-                    wallet_name={wallet_name}
                 />
-            </Modal>
-        </div>
-    );
-};
+
+                {is_mobile && <div id='mobile_list_modal_root' />}
+
+                <Modal
+                    className='transfer-account-selector__modal-header'
+                    has_close_icon
+                    is_open={is_list_modal_open}
+                    portalId={portal_id}
+                    title={label}
+                    toggleModal={() => setIsListModalOpen(old => !old)}
+                    min_height={is_mobile ? mobile_list_min_height : ''}
+                >
+                    <TransferAccountList
+                        is_mobile={is_mobile}
+                        selected_account={selected_account}
+                        setIsListModalOpen={setIsListModalOpen}
+                        setSelectedAccount={setSelectedAccount}
+                        transfer_accounts={transfer_accounts}
+                        transfer_hint={transfer_hint}
+                        wallet_name={wallet_name}
+                    />
+                </Modal>
+            </div>
+        );
+    }
+);
+
+TransferAccountSelector.displayName = 'TransferAccountSelector';
```

Comment:
@heorhi-deriv same here 🙏🏻 

By the way, do we really need to use `React.memo`? 🤔

Reply:
@farzin-deriv i removed React.memo for TransferAccountSlector, but we need it for TransferTile, because when we type the amount into input, on every keyboard press we rerender TransferTile. And when we do this, currency icon is blinking. To avoid this i added React.Memo to TransferTile.